services/handlers/dummy.ts

```diff
@@ -0,0 +1,12 @@
+import { resolverWithAuth } from './authentication'
```

Comment:
Remove dummy endpoint, I this is not required other then personal testing.

Reply:
removed.